app/views/gkr_keyword_comparison/show.html.erb

```diff
@@ -13,5 +13,18 @@
 <% end %>
 <br>
 <br>
-<h1 style='color: white'> Abstract: </h1>
-<p style='color: white'> <%= @abstract %> </p>
\ No newline at end of file
+
+<% if @abstract != nil then %>
```

Comment:
you can remove `!= nil then` and your code will function the same. It is rails convention to only use `then` for inline ternaries so you'll want to remove `then`. Also, if you want to do the `nil` check you can (though I personally don't like it) you should use `unless @abstract.nil?` instead of `!= nil` as it's rails best practice to use the rails built-ins